include/dnscpp/lookup.h

```diff
@@ -31,7 +32,7 @@ namespace DNS {
 /**
  *  Class definition
  */
-class Lookup : public Operation
+class Lookup : public Operation, public IntrusiveQueueItem<Lookup>
```

Comment:
The lookup should be agnostic for the container in which it is stored (it should be just as easy to store it in a set, map, vector or whatever, and should not be tied to an "IntrusiveQueue"